src/buildHTML.js

```diff
@@ -140,6 +141,8 @@ const shouldHandleSupSub = function(group, options) {
             group.value.alwaysHandleSupSub);
     } else if (group.type === "accent") {
         return isCharacterBox(group.value.base);
+    } else if (group.type === "horizBrace") {
+        return (isSup === group.value.isOver);
```

Comment:
That seems reasonable since we're always passing `group.value.base` at all of the call sites.